WORKSPACE

```diff
@@ -65,7 +65,6 @@ dpkg_list(
         "libffi6",
         "libncursesw5",
         "liblzma5",
-        "libexpat1",
```

Comment:
libexpat1 is also used by the python builds, so maybe this line should stay here.

Reply:
Will revert. (I thought I could ignore those python builds because they were under `/experimental`, but now I think perhaps `/experimental` is part of an actual build.)